Exclude the __pycache__ directory or pyc files from linting

**Is your feature request related to a problem? Please describe.**
super-linter includes the `__pycache__` directory (and `.pyc` files) when building the file list. Linters that are set to check every file, such as `editorconfig-checker`, fail on Python bytecode files.

**Describe the solution you'd like**
Ignore the `__pycache__` directory, or `*.pyc` files

**Additional context**

Example:

```shell
2020-11-13 13:04:38 [ERROR]   Found errors in [editorconfig-checker] linter!
2020-11-13 13:04:38 [ERROR]   Error code: 1. Command run: whatever/__pycache__/__init__.cpython-38.pyc:
	Wrong line endings or new final newline
	Not all lines have the correct end of line character
```

--- a/lib/functions/buildFileList.sh
+++ b/lib/functions/buildFileList.sh
@@ -95,9 +95,11 @@ function BuildFileList() {
     debug "Populating the file list with all the files in the ${WORKSPACE_PATH} workspace"
     mapfile -t RAW_FILE_ARRAY < <(find "${WORKSPACE_PATH}" \
       -not \( -path '*/\.git' -prune \) \
+      -not \( -path '*/\.pytest_cache' -prune \) \
       -not \( -path '*/\.rbenv' -prune \) \
       -not \( -path '*/\.terragrunt-cache' -prune \) \
       -not \( -path '*/\.venv' -prune \) \
+      -not \( -path '*/\__pycache__' -prune \) \
       -type f \
       2>&1 | sort)
 

